tools/jil/driver/index.js

```diff
@@ -31,8 +31,8 @@ wd.addAsyncMethod('safeGet', function (url, cb) {
 
 wd.addAsyncMethod('waitForFeature', function (feat, cb) {
   this.waitFor(
-    asserters.jsCondition('window.NREUM && window.NREUM.setToken && window.NREUM.setToken.active && window.NREUM.setToken.active.err'),
-    // asserters.jsCondition('window.NREUM && window.NREUM.activatedFeatures && window.NREUM.activatedFeatures.err'),
+    // asserters.jsCondition('window.NREUM && window.NREUM.setToken && window.NREUM.setToken.active && window.NREUM.setToken.active.err'),
+    asserters.jsCondition(`window.NREUM && window.NREUM.activatedFeatures && window.NREUM.activatedFeatures['${feat}']`),
```

Comment:
This is one important bit that was blocking JIL harvests from fulfilling with the new modular code.